sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawPod.java

```diff
@@ -29,6 +29,7 @@
     private final String preReservedRole;
     private final WriteOnceLinkedHashMap<String, RawSecret> secrets;
     private final Boolean sharePidNamespace;
+    private final Boolean allowDecommission;
```

Comment:
Should we initialise this to `false` explicitly?

Reply:
Eh I was just reusing the current `sharePidNamespace` behavior here